lib/dbservice/sessions.ex

```diff
@@ -102,6 +102,23 @@ defmodule Dbservice.Sessions do
     Session.changeset(session, attrs)
   end
 
+  @doc """
+  Updates the users mapped to a batch.
+  """
+  def update_batches(session_id, batch_ids) when is_list(batch_ids) do
```

Comment:
There's no user anywhere in this method, though users are mentioned in the doc. This is intentional?

Reply:
Sorry copy/paste error. I'll update.